### DISCUSSION TITLE
안녕하세요 머신러닝 처음 사용자입니다.

### DISCUSSION THREAD

**Discussion by ba5909** [reinforcement-learning-kr/2021_RLKR_Drone_Delivery_Challenge_with_Unity#7] [General]
안녕하세요 머신러닝 처음 사용자입니다.

유니티는 자주 쓰다보니 유튜브 보면서 ml-agents를 따라 해보는 건 어렵지 않았습니다.
에디터 상에서 예제 씬 몇 번 돌려보면 금방 성장하는게 보여서 재밌기도 했고요.

하지만 이번 챌린지는 완성된 빌드 파일로 학습시키는 거라 그런지 따라 할만한 자료를 찾기도 어렵고
결과물 확인도 제가 제대로 하는게 맞는지 모르겠어요.

3일동안 --resume으로 끊어가며 max_steps 2000만 정도 학습 시켰는데 성장이 멈춘 것 같아요.
파라메터는 뭘 건드려야 할 지 몰라서 일단 디폴트로 진행했습니다.
tensorboard --logdir results로 그래프 확인하면 600만회부터 결과가 비슷비슷한 것 같습니다.

근데 막상 --inference로 실행해서 보면 처음 돌렸을때와 비슷하게 보여서 제가 제대로 확인하는게 맞는지 의심이 생깁니다.
디렉토리 구조는 예시 환경과 비슷하게 맞췄고
러닝 확인할때는
mlagents-learn ./drone_config/ppo.yaml --env=./RLVillage --resume --run-id=firstRun --inference --width=576 --height=324
이어서 러닝할때는
mlagents-learn ./drone_config/ppo.yaml --env=./RLVillage --run-id=firstRun --resume --num-envs=5 --no-graphics
이런식으로 했어요

이렇게 하는게 맞는 걸까요?
파라메터 값을 바꿔서 새로 진행하는게 나을까요??
경험 있으신 분들께 도움 바랍니다.

감사합니다.

**Comment by pnltoen** [comment]
저 같은 경우는 지금까지 시간이 지남에 따라 결과가 더 나아지지않거나 오히려 나빠질 경우 파라미터 값을 수정해서 진행하였습니다! tensorboard로 확인하고 진행하시면 좋을 것 같아요!
